check_rpmspec_collection.sh

```diff
@@ -41,11 +42,11 @@ run() {
 }
 
 run "--without ansible" "without_ansible" 1
-run "--without ansible --with collection_artifact" "without_ansible-with_collection_artifact" 1
+run "--without ansible --with collection_artifact" "without_ansible-with_collection_artifact" 0
 run "--without ansible --without collection_artifact" "without_ansible-without_collection_artifact" 1
 run "--without collection_artifact" "without_collection_artifact" 1
-run "--with ansbile --without collection_artifact" "with_ansible-without_collection_artifact" 1
+run "--with ansible --without collection_artifact" "with_ansible-without_collection_artifact" 1
 run "" "no_opt" 1
-run "--with ansbile" "with_ansible" 1
+run "--with Ansible" "with_ansible" 1
```

Comment:
capital `A`?

Reply:
*grrr...* Sorry, I mistakenly pressed a wrong number in `aspell check`... (;_;)